pkg/operator/controller/canary/daemonset.go

```diff
@@ -53,6 +62,21 @@ func (r *reconciler) createCanaryDaemonSet(daemonset *appsv1.DaemonSet) error {
 	return nil
 }
 
+// updatedCanaryDaemonSet updates the canary daemonset if an appropriate change
```

Comment:
```suggestion
// updateCanaryDaemonSet updates the canary daemonset if an appropriate change
```
But we can get that in a follow-up.

Reply:
So close!